<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientPage</class>
 <widget class="QDialog" name="PatientPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Information About Patient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>New Appointment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>291</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Optima</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Patient Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Optima</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Doctor ID: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Optima</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_PatientName">
        <property name="text">
         <string>PatientName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_DoctorID">
        <property name="text">
         <string>Doctor ID: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_PhoneNumber">
        <property name="text">
         <string>PhoneNumber</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_SocialNumber">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>119</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>159</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Socialnr: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
